t.sentinel.import/t.sentinel.import.py

```diff
@@ -496,8 +533,41 @@ def main():
             cloudlist.append(vect)
             grass.run_command('g.copy', vector=vect + '@' + new_mapset + ',' + vect)
         for rast in grass.parse_command('g.list', type='raster', mapset=new_mapset):
-            maplist.append(rast)
-            grass.run_command('g.copy', raster=rast + '@' + new_mapset + ',' + rast)
+            grass.run_command('g.copy',
+                              raster=rast + '@' + new_mapset + ',' + rast)
+            if "CLOUDS" in rast:
+                cloudlist.append(rast)
+            else:
+                maplist.append(rast)
+
+                if options["offset"]:
+                    # save the description.json
+                    tmp_desc_dir = os.path.join(tmpdirectory, "descriptions_json")
+                    if not os.path.isdir(tmp_desc_dir):
+                        try:
+                            os.makedirs(tmp_desc_dir)
+                        except:
+                            grass.fatal(_(f"Unable to create directory {tmp_desc_dir}"))
+
+                    desc_file_save = os.path.join(tmp_desc_dir, f"{rast}_description.json")
+                    desc_file_in = os.path.join(json_standard_folder, rast, "description.json")
+                    shutil.copy(desc_file_in, desc_file_save)
+
+                    # calculate offset (metadata in cell_misc will be lost)
+                    tmp_rast = f"rast_tmp_{os.getpid()}"
+                    mapc_exp = (f"{tmp_rast} = if({rast} + {options['offset']} < 0, "
+                                f"0, {rast} + {options['offset']} )")
```

Comment:
Why is the if-statement or rather the clipping to 0 explicitely needed here? Maybe also add as short comment

Reply:
It is not a hard requirement, just minor cosmetics to avoid negative values. I added a comment accordingly